### PR TITLE
feat(content): history

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## 🍰 Pullrequest

Start to work on the history of IT4C

### Todo
- [x] finish the history doc

### CODE REVIEW

File: docs/history.md [hunk 1]
```diff
@@ -1,4 +1,30 @@
...
\ No newline at end of file
+This section gives an overview when which person became a member of our team and how IT4C became to be.
+
+It all began ~2017, when [Alex](people/alexander-friedland.md), who made the initial website for the DEMOCRACY project, met with [Ulf](people/ulf-gebhardt.md) in Karlsruhe to <!-- textlint-disable write-good-->evaluate<!-- textlint-enable write-good --> the state of election methods available in universities at given time. Ulf started to work with DEMOCRACY and when the project had its public appearance [Wolle](people/wolfgang-huss.md) made contact to work with the project as an influencer. 
+
+## Human Connection
+
+Most of us met again 2019 at Human Connection. This time all as developers to <!-- textlint-disable write-good-->implement<!-- textlint-enable write-good --> the envisioned social network. [Wolle](people/wolfgang-huss.md) was first, followed by [Alex](people/alexander-friedland.md) and [Ulf](people/ulf-gebhardt.md). While Ulf left for DEMOCRACY again, Hannes volunteered and Moriz joined the Team, till the project ended 2020 as Human Connection and was transformed into Ocelot.social.
```
Comment (Mogge) [inline docs/history.md:9]: Links for Hannes and Moriz?

File: docs/history.md [hunk 2]
```diff
@@ -1,4 +1,30 @@
...
+In 2021 most of the team joined the Gradido project. As always our scout [Alex](people/alexander-friedland.md) was first, followed by [Ulf](people/ulf-gebhardt.md), who was residing in Norway at the time. Soon after Moriz joined up and Hannes came on board in Summer 2021.
+When Ulf went on a trip to Italy [Wolle](people/wolfgang-huss.md) could <!-- textlint-disable write-good-->finally<!-- textlint-enable write-good --> be convinced to join, soon followed by [Mathias](people/mathias-lenz.md).
+
+## Yunite.me
+
+During 2022 the Yunite-Team approached the team to get a new social network going based on ocelot-social. The new network was release on 01.11.2022 under the name yunite.me. 
+
+## Trouble at Gradido
+
+With the end of the year 2022 the financial situation within the Gradido project became more dire, which lead to reduced hours for the team and [Wolle](people/wolfgang-huss.md) becoming a volunteer rather then a paid contributor.
```
Comment (Mogge) [inline docs/history.md:26]: I am not sure if things like this should be published

**Review by Mogge — CHANGES_REQUESTED** [review]

File: docs/history.md [hunk 3]
```diff
@@ -1,4 +1,30 @@
...
+In the last quater of 2020 Human Connection was ended as a project and the source code was republished under the name Ocelot.social. [Wolle](people/wolfgang-huss.md), [Alex](people/alexander-friedland.md), [Ulf](people/ulf-gebhardt.md) and Moriz continued on the project and released wir.social to keep ocelot alive.
+
+## Gradido
+
+In 2021 most of the team joined the Gradido project. As always our scout [Alex](people/alexander-friedland.md) was first, followed by [Ulf](people/ulf-gebhardt.md), who was residing in Norway at the time. Soon after Moriz joined up and Hannes came on board in Summer 2021.
+When Ulf went on a trip to Italy [Wolle](people/wolfgang-huss.md) could <!-- textlint-disable write-good-->finally<!-- textlint-enable write-good --> be convinced to join, soon followed by [Mathias](people/mathias-lenz.md).
+
+## Yunite.me
+
+During 2022 the Yunite-Team approached the team to get a new social network going based on ocelot-social. The new network was release on 01.11.2022 under the name yunite.me.
```
Comment (Tirokk) [inline docs/history.md:22]: Maybe a link to yunite.me?

```suggestion
During 2022 the Yunite-Team approached the team to get a new social network going based on *ocelot.social*. The new network was release on 01.11.2022 under the name [yunite.me](https://yunite.me).
```

File: docs/history.md [hunk 4]
```diff
@@ -1,4 +1,30 @@
...
+
+Most of us met again 2019 at Human Connection. This time all as developers to <!-- textlint-disable write-good-->implement<!-- textlint-enable write-good --> the envisioned social network. [Wolle](people/wolfgang-huss.md) was first, followed by [Alex](people/alexander-friedland.md) and [Ulf](people/ulf-gebhardt.md). While Ulf left for DEMOCRACY again, Hannes volunteered and Moriz joined the Team, till the project ended 2020 as Human Connection and was transformed into Ocelot.social.
+
+## Ocelot.social - end of Human Connection
+
+In the last quater of 2020 Human Connection was ended as a project and the source code was republished under the name Ocelot.social. [Wolle](people/wolfgang-huss.md), [Alex](people/alexander-friedland.md), [Ulf](people/ulf-gebhardt.md) and Moriz continued on the project and released wir.social to keep ocelot alive.
+
+## Gradido
+
+In 2021 most of the team joined the Gradido project. As always our scout [Alex](people/alexander-friedland.md) was first, followed by [Ulf](people/ulf-gebhardt.md), who was residing in Norway at the time. Soon after Moriz joined up and Hannes came on board in Summer 2021.
```
Comment (Tirokk) [inline docs/history.md:17]: Maybe a link to Gradido?
Or to the project description here?

```suggestion
In 2021 most of the team joined the [Gradido](https://gdd.gradido.net) project. As always our scout [Alex](people/alexander-friedland.md) was first, followed by [Ulf](people/ulf-gebhardt.md), who was residing in Norway at the time. Soon after Moriz joined up and Hannes came on board in Summer 2021.
```

File: docs/history.md [hunk 5]
```diff
@@ -1,4 +1,30 @@
...
+
+During 2022 the Yunite-Team approached the team to get a new social network going based on ocelot-social. The new network was release on 01.11.2022 under the name yunite.me. 
+
+## Trouble at Gradido
+
+With the end of the year 2022 the financial situation within the Gradido project became more dire, which lead to reduced hours for the team and [Wolle](people/wolfgang-huss.md) becoming a volunteer rather then a paid contributor.
+
+## IT4C
+
+With the start of 2023 the IT4C brand name and [website](https://www.it4c.dev) was established to allow the whole team to present themselves and the team as a whole.
```
Comment (Tirokk) [inline docs/history.md:30]:
```suggestion
With the start of 2023 the IT4C brand name and [website](https://www.it4c.dev) was established to allow the team members to present themselves and the team as a whole.

```

**Review by Tirokk — CHANGES_REQUESTED** [review]
Cool cool 😍

See my suggestions and questions …

File: docs/history.md (hunk 1 above)
Comment (ulfgebhardt) [reply docs/history.md]: Yes ofc. At the given time there was no Hannes or Moriz pages yet.

File: docs/history.md (hunk 2 above)
Comment (mahula) [reply docs/history.md:26]: Yes, they should not.

File: docs/history.md (hunk 4 above)
Comment (mahula) [reply docs/history.md:17]: We should use the gradido software repo URL here.

File: docs/history.md (hunk 1 above)
Comment (mahula) [reply docs/history.md]: Fixed in 9abb6ccf1df7a2620401777542f4838fb65eabf0

File: docs/history.md (hunk 4 above)
Comment (mahula) [reply docs/history.md]: Gradido software repo URL is now inserted - 80b3c7e8f8850b49a6ca10fbdf60da0afd78edce

File: docs/history.md [hunk 6]
```diff
@@ -1,4 +1,30 @@
...
+
+## Human Connection
+
+Most of us met again 2019 at Human Connection. This time all as developers to <!-- textlint-disable write-good-->implement<!-- textlint-enable write-good --> the envisioned social network. [Wolle](people/wolfgang-huss.md) was first, followed by [Alex](people/alexander-friedland.md) and [Ulf](people/ulf-gebhardt.md). While Ulf left for DEMOCRACY again, Hannes volunteered and Moriz joined the Team, till the project ended 2020 as Human Connection and was transformed into Ocelot.social.
+
+## Ocelot.social - end of Human Connection
+
+In the last quater of 2020 Human Connection was ended as a project and the source code was republished under the name Ocelot.social. [Wolle](people/wolfgang-huss.md), [Alex](people/alexander-friedland.md), [Ulf](people/ulf-gebhardt.md) and Moriz continued on the project and released wir.social to keep ocelot alive.
```
[the remark this answers is not in the document]
Comment (Tirokk) [reply docs/history.md]: done

File: docs/history.md (hunk 3 above)
Comment (Tirokk) [reply docs/history.md]: Done

**Review by Tirokk — APPROVED** [review]
I approve

**Pull request merged**